packages/sanity/src/core/components/StatusButton.tsx

```diff
@@ -48,7 +55,7 @@ export const StatusButton = forwardRef(function StatusButton(
     <StyledButton
       data-ui="StatusButton"
       {...restProps}
-      aria-label={label || t('status-button.aria-label')}
+      aria-label={label}
```

Comment:
Maybe I'm misunderstanding, but in this case wouldn't it be wise to still have a fall back on the StatusButton itself (outside of making sure that the labels exist where the StatusButton is used?)

Reply:
Would a fallback be necessary given that the `aria-label` is now a required prop?

Reply:
Oh yeah that's true! 🤔 
I guess my follow up question is whether this should be something that we should require of all buttons then (if we don't already). Since it might be strange that we're only requiring this on a StatusButton vs everywhere else.

But maybe it's a question for another time and out of scope for this PR? 🤔

Reply:
That's a fair point! I'm not sure the situation with all other buttons but I can take a look after this PR. I'm not sure the exact process that surfaced this particular status button issue, but perhaps this was seem most commonly for now